doc/redirects.json

```
@@ -1533,9 +1533,6 @@
   "tester-shellcheck-inputs": [
     "index.html#tester-shellcheck-inputs"
   ],
-  "tester-shellcheck-param-src": [
-    "index.html#tester-shellcheck-param-src"
-  ],
```

Unsure why the id was removed, but if we're going ahead with that, it would be good to point the now removed id to its parent so that the links to this id don't break

```suggestion
  "tester-shellcheck-param-src": [
    "index.html#tester-shellcheck-inputs"
    "index.html#tester-shellcheck-param-src"
  ],
```

---

We (@philiptaron and I) were unable to get this to work -- adding it as a key caused the nix manual to fail to build, so we settled on just breaking the link, because this particular instance was irregular and arguably shouldn't have been added in the first place.